spec/support/oauth_helper.rb

```diff
@@ -4,11 +4,11 @@ def login_with_oauth(user = create(:user))
     visit "/auth/githubteammember"
   end
 
-  def setup_mock_auth(email = "test@example.com")
+  def setup_mock_auth(email = "test@example.com", options = {})
     OmniAuth.config.mock_auth[:githubteammember] =
       OmniAuth::AuthHash.new(
         credentials: {
-          "team_member?" => true,
+          "team_member?" => options[:team_member].nil? ? true : options[:team_member],
```

Comment:
we can change this in the future (no need to do it now), but I generally avoid these types of ternary statements. Reason: They are nice and compact but can be harder to understand.